packages/rscode/src/extension.ts

```diff
@@ -179,6 +181,80 @@ ${text}
     }
   );
 
+    // Add Docs command
+    const add_docs = vscode.commands.registerTextEditorCommand(
+      'extension.rocksetAdd',
+      async (activeEditor) => {
+        try { // try to parse JSON
+          const docs = JSON.parse(activeEditor.document.getText());
+          try {
+            client.workspaces.listWorkspaces().then(function(raw_workspaces) { // list workspaces
+              let workspaces: string[] = []
+              
+              raw_workspaces.data?.forEach(item => { // for each workspace, add it's name to `workspaces`
+                if (typeof(item.name) == "string"){ workspaces.push(item.name) }
+              });
+              vscode.window.showQuickPick(workspaces, { placeHolder: "workspace" }).then(workspace => { // show dropdown menu of workspaces
+                if (!workspace) { return } // if user exits, return
+
+                client.collections.workspaceCollections(workspace).then(function(raw_collections){ // list collections in workspace
+                  let collections: string[] = []
+                  raw_collections.data?.forEach(item => {
+                    if (typeof(item.name) == "string") { collections.push(item.name) }
```

Comment:
this should be ===

https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a